src/consensus/aft/impl/view_change_tracker.h

```diff
@@ -258,6 +258,7 @@ namespace aft
         nv.view_change_messages.emplace(it.first, it.second);
       }
 
+      store->sign_view_change_confirmation(nv);
```

Comment:
Was this not signed at all before, or is this code being moved from somewhere else?

Reply:
yeah not signed before, so fun bug here :S